src/index.js

```diff
@@ -1,93 +1,68 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import '@babel/polyfill';
+import errorhandler from 'errorhandler';
+import express, { json, urlencoded } from 'express';
+import { config } from 'dotenv';
+import cors from 'cors';
+import Debug from 'debug';
+import routes from './routes';
 
-const isProduction = process.env.NODE_ENV === "production";
+config();
 
-// Create global app object
-const app = express();
+const debug = Debug('dev');
 
+const { PORT } = process.env; // setup port to be used
```

Comment:
Can you please remove all the comments on lines 13, 14, 26, 37, 39, 40, 54 and 55?